lib/app.js

```diff
@@ -49,6 +49,65 @@ app.get('/animals/:id', async (req, res) => {
   }
 });
 
+
+app.post('/animals', async (req, res) => {
+  try {
+    const data = await client.query(`
+    INSERT into animals ( id, name, type, snuggly)
+    VALUES ($1, $2, $3, $4)
+    RETURNING *`
+    , [req.body.id, req.body.name, req.body.type, req.body.snuggly]);
+    res.json(data.rows[0]);
+  } catch (e) {
+    res.status(500).json({ error: e.message });
+  }
+});
+
+//took out id 
+app.put('/animals/:id', async (req, res) => {
+  try {
+    const data = await client.query(`
+    UPDATE animals
+    SET 
+    name=$2,
+    type=$3,
+    snuggly=$4
+    WHERE id=$1
+    RETURNING *
+`
+    , [req.params.id, req.body.name, req.body.type, req.body.snuggly]);
+    res.json(data.rows[0]);
+  } catch (e) {
+    res.status(500).json({ error: e.message });
+  }
+});
+
+app.post('/animals/:id', async (req, res) => {
```

Comment:
You don't need this route since it essentially does the same thing as the other post route. And typically a user won't be supplying the id when adding something new (it's something sql will do automatically) so /id wouldn't make sense to have for a post route